### PR TITLE
SCSS - Menu style cleanup

### PR DESCRIPTION
**Pull request by frogpad7**
Returning the black border to the Official Banner
&
Reverting the interaction behavior of the Sidebar Button of how it looked previously

### CODE REVIEW

**Comment by frogpad7** [comment]
It seems as if the color given to the Sidebar Button could just have an override flag to return back to what it was previously (with this: @extends .bg-white). What should the end result of this new SCSS file consist of?

File: scss/_sidebar-button.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.sidebar-button {
+  @extend .position-relative;
+  @extend .btn;
+  @extend .btn-link;
+  color: $white;
+}
```
Comment (isralduke) [inline scss/_sidebar-button.scss:6]: Let’s add a blank line at the end of this file @frogpad7

**Review by isralduke — CHANGES_REQUESTED** [review]
One comment

**Review by isralduke — APPROVED** [review]
Looks good

**Pull request merged**